apps/prairielearn/src/pages/instructorAssessmentGroups/instructorAssessmentGroups.js

```diff
@@ -149,43 +138,25 @@ router.post(
       const group_id = req.body.group_id;
       const uids = req.body.add_member_uids;
       const uidlist = uids.split(/[ ,]+/).filter((uid) => !!uid);
-      const failedUids = [];
-      const duplicateUids = [];
       for (const uid of uidlist) {
-        // Check if the user is already in another group for this assessment.
-        // If so, we'll display a special error message for them.
-        // If we can't find a user for a given UID, fall through to the sproc,
-        // which will handle this case.
         const user = await selectUserByUid(uid);
-        if (user) {
-          const existingGroupId = await getGroupId(assessment_id, user.user_id);
-          if (existingGroupId != null && !idsEqual(existingGroupId, group_id)) {
-            duplicateUids.push(uid);
-            continue;
-          }
+        if (!user) {
+          flash('error', `Failed to add the user ${uid}: User does not exist.`);
+          continue;
         }
 
-        let params = [assessment_id, group_id, uid, res.locals.authn_user.user_id];
         try {
-          await sqldb.callAsync('assessment_groups_add_member', params);
+          await addUserToGroup(
+            assessment_id,
+            group_id,
+            user.user_id,
+            res.locals.authn_user.user_id,
+            false, // Enforce group size limits (instructors can override limits)
```

Comment:
Rather than a plain boolean, consider an options object that can be set like `{ enforceSizeLimits: false }`. In PT we've leaned into functions that take 1-2 objects with named properties instead of taking 5-6 separate arguments. We figure this is the best we can do given JS's lack of named parameters.